### DISCUSSION TITLE
How to import Rest methods of AMS to Postman easily?

### DISCUSSION THREAD

**Discussion by yashtandon113** [ant-media/Ant-Media-Server#5664] [FAQs]
How to import all Rest API methods of Ant Media Server to Postman easily?

**Comment by yashtandon113** [comment]
Hi Guys,

There is a simple way of importing all of the rest methods from Swagger to Postman. Here is a step-by-step guide:

1. Go to https://antmedia.io/rest/ and select the Ant Media Server version you are using on the top right-hand corner.

![image](https://github.com/ant-media/Ant-Media-Server/assets/82374739/e066e1b6-4fb9-42a4-9b23-db29a90d8587)

2. Copy the URL of the swagger.json file.

![image](https://github.com/ant-media/Ant-Media-Server/assets/82374739/ea80a9be-41ca-4624-9f7a-849ab7067abc)

3. Open Postman and click the import button.

![image](https://github.com/ant-media/Ant-Media-Server/assets/82374739/4980fa7a-e241-4f1b-8ae7-ed3b49569603)

4. Go to the link section and paste the URL of the swagger.json file copied before.

![image](https://github.com/ant-media/Ant-Media-Server/assets/82374739/904856d8-a6b4-4dda-a109-c6ad45eb94e1)

5. Click on the Ant Media Server REST API Reference collection and go to the Variables section.

![image](https://github.com/ant-media/Ant-Media-Server/assets/82374739/41993014-4f00-4f7c-8611-384b182cfc5d)

Change baseUrl with your domain name.

****Note:** Current value should be like**  `https://domain:5443/Application_name/rest`


Congrats! You are ready to call all Rest APIs of AMS from a Postman without adding it one by one 🙂 

​

**Reply by swathichattu** [reply]
Is There any Suggestions, To check  VOD(video on Demand) REST API's in Postman.Main Thing I don't want to Import All API's. I am going to Create  a POC(Proof of Concepts) on VOD ,Suggest me if it has anything important for that. Thank You

**Comment by lastpeony** [comment]
This is incredibly useful. Thanks for sharing, Yash!

**Reply by yashtandon113** [reply]
Thanks to @mustafaboleken as he posted this previously on old [community platform](https://community.antmedia.io). So I thought to put it here for the users :)
